containers/allan_dip_bot/BUILD.md

```diff
@@ -22,6 +22,6 @@ $ docker run -it allan_dip_bot --help
 --outdir    OUT_DIR
 
 #connect to remote game engine
-$ docker run -it allan_dip_bot --game_id test_game --host shade.tacc.utexas.edu --power TURKEY
+$ docker run -it allan_dip_bot --game_id test_game --host shade-dev.tacc.utexas.edu --power TURKEY
```

Comment:
Not in this PR but why is this hard coded to Turkey?

Reply:
This is just an example command in a documentation file.